### ISSUE TITLE
[9.0] Discuss not showing messages

### ISSUE THREAD

**Issue by adekock11** [odoo/odoo#10559]
I am currently on the Odoo Discuss menu, inside the #general channel.

Current behaviour:
1) I type a message in the #general channel
2) I press enter, but the message does not appear in the history
3) When I refresh the browser the message is now visible in the history

Expected behaviour:
1) I type a message in the #general channel
![01](https://cloud.githubusercontent.com/assets/1809451/12456185/58b2fe0c-bfa7-11e5-8423-b7c334bb7575.JPG)
2) I press enter and the message appears
![02](https://cloud.githubusercontent.com/assets/1809451/12456200/64b6d4b2-bfa7-11e5-9d57-288d34e4fe3f.JPG)

**Comment by lukebranch** [comment]
@adekock11 ,

This is because you do not have psycogreen (dependency for the longpolling/chat function) installed.

This means livechat will not function.

Dependency is installed by pip:

sudo pip install psycogreen

**Comment by adekock11** [comment]
Hi @lukebranch. Correct, one needs to install that dependency. My mistake was in my Nginx configuration where I forwarded the location /longpolling on port 80 to 127.0.0.1:8071 instead of 127.0.0.1:8072. Now the chat works with the port set to 8072, so I closed this issue.

One thing that puzzles me is on Odoo's official docker file [1] one can clearly see that they open port 8071 for longpolling. However, on my instance the default longpolling port is 8072. That is where I got confused. However in the default documentation [2] I can see that the default longpolling port is in fact 8072.

[1] https://github.com/odoo/docker/blob/master/9.0/Dockerfile#L44
[2] https://www.odoo.com/documentation/8.0/reference/cmdline.html
